package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mia-platform/crud-service",
-  "version": "6.9.6",
+  "version": "6.9.7-rc.0",
```

Comment:
Can you revert the commit that created the tag before to merge this feature?

Reply:
I will update this when I'll create the next tag.